aip/general/0121.md

```diff
@@ -81,6 +81,27 @@ In an API with a stateless protocol, the server has the responsibility for
 persisting data, which may be shared between multiple clients, while clients
 have sole responsibility and authority for maintaining the application state.
 
+### Cyclic References
+
+The relationship between resources, such as parent-child or
+[resource references][], **must** be representable via an acylic graph.
+
+A cyclic relationship between resources increases the complexity of managing
+resources significantly more complex. Consider a resource A and B that refer to
+each other. The process to create said resources are:
+
+1. create resource A without a reference to B. Retrieve id for resource A.
+2. create resource B with a reference to A. Retrieve id for resource B.
+3. update resource A with the reference to B.
+
+The delete operation may also become more complex, due to reasoning about which resource must be dereferenced first for a successful deletion.
+
+This requirement does not apply to relationships that are expressed via
+[output only][] fields. Output only fields with cyclic references can be
+useful to help reduce the need to retrieve multiple resources to understand the relationship, and do not increase complexity in the management of resources as
```

Comment:
Personally, I don't think we need to explain why output only cyclic references could be useful in this section - that sounds like a separate guidance/AIP. I would just take out the entire middle part "can be
useful to help reduce the need to retrieve multiple resources to understand the relationship," and say "Output only fields with cyclic references do not increase complexity...."

Reply:
ack, sounds good.